src/hivpy/hiv_status.py

```diff
@@ -82,6 +91,20 @@ def __init__(self):
         self.cm_mortality_factor = rng.choice([3, 5, 10])
         self.other_adc_mortality_factor = rng.choice([1.5, 2, 3])
 
+        self.hiv_test_type = HIVTestType.Ab
+        self.test_sens_general = 0.98
+        self.test_sens_primary_ab = rng.choice([0.5, 0.75])
+        self.test_sens_prep_inj_primary_ab = rng.choice([0, 0.1])
+        # based on sens_tests_prep_inj in the SAS code
+        self.test_sens_prep_inj_primary_pcr = rng.choice([0.7, 0.5, 0.3, 0.2])
+        # FIXME: double-check if 0.05 being present twice is intentional
+        self.prob_loss_at_diag = rng.choice([0.01, 0.02, 0.04, 0.05, 0.05, 0.15, 0.30, 0.35, 0.50, 0.60])
```

Comment:
This is intentional because (0.02, 0.05, 0.15, 0.35, 0.50) are the values and (0.60, 0.30, 0.05, 0.04, 0.01) are their respective sample probabilities.